src/components/error-summary/index.md.njk

```diff
@@ -36,6 +36,22 @@ There are 2 ways to use the error summary component. You can use HTML or, if you
 
 {{ example({group: "components", item: "error-summary", example: "default", html: true, nunjucks: true, open: false, size: "s", id: "default-2"}) }}
 
+### Linking from the error summary to each question
+
+You must link the errors in the error summary to the question they relate to.
+
+For questions that use a single field, for example the character count, file upload, select, textarea or text input components, link to the field.
```

Comment:
```suggestion
Where a user has to answer using a single field, for example, a character count, file upload, select, textarea or text input components, link to the field.
```

Reply:
I don't think that's _quite_ right, because it suggests that fields == components which is not right. Should we also use 'form controls' here?

```suggestion
Where a user answers using a single form control, for example, if you are using a character count, file upload, select, textarea or text input component, link to the field.
```